src/main/java/com/github/lipinskipawel/board/ai/bruteforce/SmartBoardEvaluator.java

```diff
@@ -0,0 +1,26 @@
+package com.github.lipinskipawel.board.ai.bruteforce;
+
+import com.github.lipinskipawel.board.ai.BoardEvaluator;
+import com.github.lipinskipawel.board.engine.BoardInterface;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toList;
+
+public class SmartBoardEvaluator implements BoardEvaluator {
+
+    private static int CONST = 10;
+    private static List<Integer> POINTS = IntStream.range(-7, 7)
```

Comment:
Is it possible to make this code simpler?
Maybe some clever usage of [%](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/op1.html)?
I can bet that the size of the board will never change, so we can actually even hardcoded this.

Reply:
This code is quite simple. I don't see simplification coming from usage of %. Maybe you have idea I don't see? For sure this code could be named better. For now I hardcoded those values. You can pick better option :)